doc/ref/domain.xml

```diff
@@ -680,8 +680,7 @@ and <Ref Filt="IsLieMatrix"/>.
 
 The following categories of elements are to be understood mainly as
 categories for all objects in a family,
-they are usually used as third argument of <C>NewFamily</C>
-(see&nbsp;<Ref Sect="Creating Families"/>).
+they are usually used as third argument of <Ref Func="NewFamily"/>.
```

Comment:
I would leave the name "NewFamily" in there, like
"... third argument of <C>NewFamily<C> (<Ref Func="NewFamily">)"
Otherwise it looks strange (e.g. "third argument of 12.32").

Reply:
Actually, no, it is rendered like this:
> they are usually used as third argument of NewFamily (13.1-2).